app/Amity.py

```diff
@@ -0,0 +1,186 @@
+    amity load_people
+    amity print_allocations
+    amity print_unallocated
+    amity print_room <room_name>
+    amity save_state[ -db =sqlite_database]
+    amity load_state <sqlite_database>
+    amity (-o | --filename)
+    amity (-i | --interactive)
+    amity (-h | --help | --version)
+
+Options:
+    -i, --interactive  Interactive Mode
+    -h, --help  Show this screen and exit.
+        --export
+"""
+
+import sys
+import cmd
+from docopt import docopt, DocoptExit
+from classes.person import Person
+from classes.room import Room
+from random import randint
+# From inventory import Item_console
+
+
+def docopt_cmd(func):
+    """
+    This decorator is used to simplify the try/except block and pass the result
+    of the docopt parsing to the called action.
+    """
+
+    def fn(self, arg):
+        try:
+            opt = docopt(fn.__doc__, arg)
+
+        except DocoptExit as e:
+            # The DocoptExit is thrown when the args do not match.
+            # We print a message to the user and the usage block.
+
+            print('Invalid Command!')
+            print(e)
+            return
+
+        except SystemExit:
+            # The SystemExit exception prints the usage for --help
+            # We do not need to do the print here.
+
+            return
+
+        return func(self, opt)
+
+    fn.__name__ = func.__name__
+    fn.__doc__ = func.__doc__
+    fn.__dict__.update(func.__dict__)
+    return fn
+
+
+class MyInteractive(cmd.Cmd):
+    intro = 'Welcome to Amity'
+    prompt = 'Tell_Amity_To>> '
+    file = None
+
+    @docopt_cmd
+    def do_create_room(self, args):
+        """
+        Usage: create_room <category> <room_name>...
+        """
+        room_names = args["<room_name>"]
+        category = args["<category>"]
+        add_obj = Room()
+
+        #print "this is to show that this function has a heart"
+
+        if category.upper() != 'LIVING' and category.upper() != 'OFFICE':
+
+            print category + ' Is not a valid Room type, \
+                    try again with office or living'
+
+        else:
+
+            add_obj.create_Rooms(room_names, category.upper())
+
+    @docopt_cmd
+    def do_add_person(self, args):
+        """
+        Usage: add_person <f_name> <s_name> <person_job> <employee_number> [<wants_accomodation>]
+        """
+        s_name =args["<s_name>"]
+        employee_number = None
+        f_name = args["<f_name>"]
+        person_job = args["<person_job>"]
+        wants_accomodation = args["<wants_accomodation>"] or "N"
+        employee_number =args["<employee_number>"]
+
+        # Create Employee number using number generator and stringbuilder
+        # number =randint(0,1000)
+        #
+        # if person_job.upper()=="FELLOW":
+        #
+        #     employee_number = 'F%d' %(number)
+        #
+        # elif person_job.upper() == "STAFF":
+        #     employee_number = 'S%d' %(number)
+
+        add_obj = Person()
```

Comment:
instead of instantiating a class at each implementation, Consider importing the functions from the class in the top level import statements to reduce repetition of Class instantiating. referrring to `add_obj = Person()`